src/front/glsl/ast.rs

```diff
@@ -178,3 +197,9 @@ pub struct FunctionCall {
     pub kind: FunctionCallKind,
     pub args: Vec<ExpressionRule>,
 }
+
+#[derive(Debug, Clone, Copy)]
+pub enum StorageQualifier {
+    StorageClass(StorageClass),
+    Const,
```

Comment:
You reminded me to file something up about this - https://github.com/gpuweb/gpuweb/issues/1394